plugin/xcodebuild.vim

```diff
@@ -0,0 +1,48 @@
+command! XcodebuildBuild call <sid>build()
+command! XcodebuildTest call <sid>test()
+
+let s:plugin_path = expand("<sfile>:p:h:h")
+
+function! s:build()
+  let cmd = s:base_command()  . s:xcpretty()
+  call s:run_command(cmd)
+endfunction
+
+function! s:test()
+  let cmd =  s:base_command() . ' -sdk iphonesimulator test' . s:xcpretty()
+  call s:run_command(cmd)
+endfunction
+
+function! s:run_command(cmd)
+  execute '!' . a:cmd
+endfunction
+
+function! s:base_command()
+  return 'xcodebuild ' . s:build_target() . ' ' . s:scheme()
+endfunction
+
+function! s:build_target()
+  let xcworkspaceFile = globpath(expand('.'), '*.xcworkspace')
```

Comment:
I think you can just use `glob('*.xcworkspace')` here. `glabpath` is for globbing accross a comma separated list of paths.